app/presenters/statistics_presenter.rb

```diff
@@ -31,7 +31,7 @@ def version
   end
 
   def open_registrations?
-    AppConfig.settings.enable_registrations
+    AppConfig.settings.enable_registration?
```

Comment:
@jhass : Nope, [see by yourself](https://travis-ci.org/diaspora/diaspora/jobs/48867657#L453). It doesn't work, plus it makes to other tests to fail. :/